task/10-katas-1-tasks.js

```diff
@@ -17,8 +17,43 @@
  *  ]
  */
 function createCompassPoints() {
-    throw new Error('Not implemented');
     var sides = ['N','E','S','W'];  // use array of cardinal directions only!
```

Comment:
Please , don't use `var` in code. It may cause an error in some cases.
We have `let , const`